draft-pauly-taps-transport-security.md

```diff
@@ -354,6 +357,16 @@ before using single-RTT handshake keys after the next TLS flight.
 - QUIC transport relies on TLS 1.3 for authentication and initial key derivation.
 - TLS within QUIC relies on a reliable stream abstraction for its handshake.
 
+## gQUIC
+
+gQUIC is a UDP-based multiplexed streaming protocol designed and deployed by Google following experience from deploying SPDY, the proprietary predecessor to HTTP/2.
+gQUIC was originally known as "QUIC": this document uses gQUIC to unambiguously distinguish it from the standards-track IETF QUIC.
+The proprietary technical forebear of IETF QUIC, gQUIC was originally designed with tightly-integrated security and application data transport protocols.
+
+### Protocol Description
+
```

Comment:
Rather than leave these blank, can you drop in something similar to the following?

((TODO: write me))